src/layman/common/util.py

```diff
@@ -0,0 +1,14 @@
+def get_names_from_infos(infos):
+    return list(set(sorted([info for info in infos])))
```

Comment:
If you need sorted, then set() makes is unsorted, sou you need to call set() first, sorted() second.

Reply:
But do you need sorted? I think not.

Reply:
The reason was, that then we can change sorted() or unsorted variant in one place, not many.